lib/saluki-components/src/sources/otlp/metrics/config.rs

```diff
@@ -122,6 +137,8 @@ impl Default for OtlpTranslatorConfig {
             instrumentation_library_metadata_as_tags: false,
             with_remapping: false,
             with_otel_prefix: false,
+            delta_ttl: Duration::from_secs(60),
+            sweep_interval: Duration::from_secs(30),
```

Comment:
Do these come from somewhere in particular, or did we just come up with our own values here?